src/main/java/teammates/storage/entity/StudentProfile.java

```diff
@@ -170,4 +174,12 @@ public void setModifiedDate(Date modifiedDate) {
         this.modifiedDate = modifiedDate;
     }
 
+    // Equalizes Objectify and GenderType enum
+    public void importGender(@AlsoLoad("gender") String gender) {
+        if (gender == null) {
+            return;
+        }
+        this.gender = Gender.valueOf(gender.toUpperCase());
```

Comment:
The parameter `gender` is not secured here. What if an invalid value is passed?

Reply:
@wkurniawan07  sir this method was made to check  for invalid gender also as this method is called from teammates.test.cases.datatransfer.StudentProfileAttributesTest and its method
1) testValidImportGender()
2) testInValidImportGender() --> as you can see that we expect that illegalArgumentException to be thrown so securing it do not give any benefit as its expected behaviour for invalid gender will change